test/client/fixtures/sandbox/code-instrumentation/setters-test.js

```diff
@@ -622,3 +622,29 @@ test('setter returns a correct value (GH-907)', function () {
     checkReturnedValue('img', 'src', './path');
     checkReturnedValue('a', 'target', '_blank');
 });
+
+test('body of new document implementation should not throw an error after set him innerHTML (GH-1172)', function () {
```

Comment:
Rename to `should not throw an error on setting the body.innerHtml when document.body equals null`